swingtix/bookkeeper/test_model.py

```diff
@@ -352,7 +350,6 @@ def test_book_set_basic(self):
             description="Expenses",
             positive_credit=False
         )
-        expense.save()
```

Comment:
Sorry should have put that in a separate PR. `save()` is redundant here as `Account.objects.create(...)` already saves the object: https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.create